Operators/Types/lib/img/generate/Raster_4f89b41b-1643-442b-bec8-9f9ef2173baa.t3ui

```diff
@@ -1,68 +1,77 @@
 {
   "Id": "4f89b41b-1643-442b-bec8-9f9ef2173baa"/*Raster*/,
-  "Description": "Generates a wide range of different raster patterns. You can also connect an input texture to drive various aspects of the pattern.",
+  "Description": "Generates a wide range of patterns of dots and lines.\n\nConnecting a texture to the Image input will allow the pattern to be influenced differently at different parts of the texture.\n\nAKA: polkadot, pattern, grid, crosshair",
```

Comment:
"The image input let the pattern be influenced by its content (reference the channel" (texture is redundant, and the passive form is a bit confusing)

Reply:
Yes, this line is very clunky in hindsight... Thanks for catching that! Will revise it (might be able to just remove it entirely in fact.)